modules/data.land/R/cohort2pool.R

```diff
@@ -83,12 +83,12 @@ cohort2pool <- function(dat, allom_param = NULL, dbh_name="DBH") {
   tot_leaf <- sum(leaf,na.rm = TRUE)
   
   #Divide by plot area, divide by 2 to convert from kg to kgC
-  leaf_biomass = ((tot_leaf/(total_area))/2 + tot_herb/1000)*1000#convert from kg to g
+  leaf_biomass = ((tot_leaf/(total_area))/2 + tot_herb)#in kg
```

Comment:
I think this fix (and the one on AGB) was done incorrectly. Specifically, I think tot_herb must have been in g. Line 73 changes the comment (already in gC) but doesn't change the units, so either the units need to be changed to kgC on line 74 or 86 and 91 need to be divided by 1000.

Finally, a reminder that explictly using PEcAn's unit conversion functions, rather than multiplying/dividing by hard-coded constants, remains best practices for the PEcAn code since it makes it much easier to see and understand unit conversions. It also reduces errors and inconsistances (e.g. different people using different numbers of significant digits during conversions)

Reply:
The tot_herb is in kg because I already converted it in the extract_NEON_veg function.